packages/frontend/src/pages/settings-page/components/profile-settings/profile-settings.tsx

```diff
@@ -183,7 +187,7 @@ export const ProfileSettings = ({ user, onSubmit, isSubmitting }: Props) => {
             <Flex direction="column" align="center">
               <FileUploadArea
                 onDrop={onDropAvatar}
-                element={<Avatar src={effectiveAvatarUrl} size="2xl" name={user.displayName} />}
+                element={<Avatar src={effectiveAvatarUrl} size={profileImgSize} name={user.displayName} />}
```

Comment:
Can't you just inline the breakpoint options since this is a Chakra UI component?

```
size={{ base: 'lg', md: '2xl' });
```

I think `useBreakpointValue` is most useful when you need to pass values to a non-Chakra component (like Ace editor)

Reply:
I did try, but it doesn't work. Seems like `size` and `variant` don't have responsive values https://chakra-ui.com/docs/hooks/use-breakpoint-value